tests/test_cookiecutter_generation.py

```diff
@@ -181,7 +181,15 @@ def test_travis_invokes_pytest(cookies, context):
 
     with open(f"{result.project}/.travis.yml", "r") as travis_yml:
         try:
-            assert yaml.load(travis_yml, Loader=yaml.FullLoader)["script"] == ["pytest"]
+            yml = yaml.load(travis_yml, Loader=yaml.FullLoader)["jobs"]["include"]
+            assert yml[0]["script"] == ["flake8"]
+
+            if context.get("use_docker") == "y":
```

Comment:
Rather than having this if branch, we can utilize [the power of parametrization](https://docs.pytest.org/en/latest/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions):

```
@pytest.mark.parametrize(["use_docker", "expected_test_script"], [
    ("n", "pytest"),
    ("y", "docker-compose -f local.yml run django pytest"),
])
def test_travis_invokes_pytest(cookies, context, use_docker, expected_test_script):
    context.update({"ci_tool": "Travis", "use_docker": use_docker})
    ...
    assert yml[1]["script"] == expected_test_script
```

Currently, I think the non-docker branch is actually not running currently. Using parametrization will also have the advantage of generating 2 separate test cases.

Ideally, we would do something similar with `test_gitlab_invokes_flake8_and_pytest`.